application/models/post_test.go

```diff
@@ -1690,16 +1690,17 @@ func (ms *ModelSuite) TestPosts_FindByUser() {
 func (ms *ModelSuite) TestPosts_GetPotentialProviders() {
 	t := ms.T()
 
-	f := createFixturesFor_Posts_GetPotentialProviders(ms)
```

Comment:
can `createFixturesFor_Posts_GetPotentialProviders` be removed now?

Reply:
:+1: